app/experimenter/nimbus-ui/src/components/FormBranches/mocks.tsx

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import FormBranches, { FormBranch } from ".";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { mockExperimentQuery } from "../../lib/mocks";
+
+// Handy mocks shared between tests and stories for this component
+
+export const SubjectBranch = ({
+  branch = MOCK_BRANCH,
+  isReference = false,
+  experimentFeatures = MOCK_EXPERIMENT_FEATURES,
+  featureSlug = "pinned_tabs",
+  featureHasValue = false,
+  equalRatio = false,
+}: {} & Partial<FormBranchProps>) => (
+  <div className="p-5">
+    <FormBranch
+      {...{
+        branch,
+        isReference,
+        experimentFeatures,
+        featureSlug,
+        featureHasValue,
+        equalRatio,
+      }}
+    />
+  </div>
+);
+
+export const SubjectBranches = ({
+  experiment = MOCK_EXPERIMENT,
+  experimentFeatures = MOCK_EXPERIMENT_FEATURES,
+  featureSlug = "pinned_tabs",
+  featureHasValue = true,
+  equalRatio = false,
+}: {} & Partial<FormBranchesProps> = {}) => (
+  <div className="p-5">
+    <FormBranches
+      {...{
+        experiment,
+        experimentFeatures,
+        equalRatio,
+        featureSlug,
+        featureHasValue,
+      }}
+    />
+  </div>
+);
+
+type FormBranchesProps = React.ComponentProps<typeof FormBranches>;
+
+type FormBranchProps = React.ComponentProps<typeof FormBranch>;
+
+export const mockExperiment = (
+  ...args: Parameters<typeof mockExperimentQuery>
+) => mockExperimentQuery(...args).data as getExperiment_experimentBySlug;
+
+export const MOCK_EXPERIMENT_FEATURES = {
+  about_welcome: { name: "about:welcome" },
+  pinned_tabs: { name: "Pinned Tabs" },
+};
+
+export const MOCK_EXPERIMENT = mockExperiment("demo-slug", {
+  treatmentBranches: [
+    {
+      __typename: "NimbusBranchType",
+      name: "Managed zero tolerance projection",
+      slug: "managed-zero-tolerance-projection",
+      description: "Next ask then he in degree order.",
+      ratio: 1,
+      featureValue: '{"effect-effect-whole": "close-teach-exactly"}',
+      featureEnabled: true,
+    },
+    {
+      __typename: "NimbusBranchType",
+      name: "Salt way link",
+      slug: "salt-way-link",
+      description: "Flame the dark true.",
+      ratio: 1,
+      featureValue: '{"frosted-wake": "simple-hesitation"}',
+      featureEnabled: true,
+    },
+  ],
+});
+
+export const MOCK_BRANCH: FormBranchProps["branch"] = MOCK_EXPERIMENT.referenceBranch!;
```

Comment:
Nice. FWIW, added a Mutation mock that might you might have thoughts on: https://github.com/mozilla/experimenter/pull/3843/files#diff-ccece61574781c5aa7869514daee9fcb76aaae79009c2ad629e034ab91261116